### ISSUE TITLE
[8.0.0-beta-3 - Nuxt 3 RC12] Unexpected i18n unplugin transformation in non-i18n-block JSON files

### ISSUE THREAD

**Issue by MusicOnline** [nuxt-modules/i18n#1603]
### Version
@nuxtjs/i18n: 8.0.0-beta-3
nuxt: 3.0.0-rc.12

### Nuxt configuration

Please change to `[x]` if relevant for this issue:

 - [ ] Applies to a site deployed to a static server (site generated with `nuxt generate`)
 - [x] Applies to a site deployed to a server with a Node backend

### Reproduction Link
https://github.com/MusicOnline/Nuxt-i18n-Unplugin-Bug-Repro

### Steps to reproduce
1. Import JSON files in any Vue SFC file in `<script setup>`

### What is Expected?
i18n unplugin does not transform imported JSON files unless it's in `<i18n src="...">` (as in beta-2)
```json
[ "a", "b", "c" ]
```

### What is actually happening?
Other imported JSON files are also transformed
```js
export default [
  (()=>{const fn=(ctx) => {const { normalize: _normalize } = ctx;return _normalize(["a"])};fn.source="a";return fn;})(),
  (()=>{const fn=(ctx) => {const { normalize: _normalize } = ctx;return _normalize(["b"])};fn.source="b";return fn;})(),
  (()=>{const fn=(ctx) => {const { normalize: _normalize } = ctx;return _normalize(["c"])};fn.source="c";return fn;})(), 
]
```

**Comment by kazupon** [comment]
I've just released edge verions `@nuxtjs/i18n-edge@8.0.0-beta.3-27791492.04f0c59`
this issue has been fixed  with edge version.
please check it!

thanks!

**Comment by MusicOnline** [comment]
Looks fixed. Thanks!
